### PR TITLE
Docker file

### PR DESCRIPTION
**Pull request by dabeat**
#### :tophat: Problem
No Docker file for development with Docker

#### :pushpin: Solution
Add it.

#### :ghost: GIF
giphy

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# image for pg-elixir-tools
```
Comment (Kitton) [inline Dockerfile]: it should be `Dockerfile.dev` I believe

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ENV MIX_ENV=prod
+ENV CODE_VERSION=$CODE_VERSION
+
+# Install hex locally
+RUN mix local.hex --force && mix local.rebar --force
+
+# Copy the contents
+COPY . /app/
+
+# Compile
```
Comment (Kitton) [inline Dockerfile]: the content of this file is a bit different from the file in `pg-payments` for example, why that?

File: Dockerfile (hunk 1 above)
Comment (dabeat) [reply Dockerfile]: Why `.dev`? In this case we don't build a Docker image in ECR, but just use it for development purposes.
As there's no gonna be the possibility of having two different Docker images for local and other environments, with the Docker file should be enough.

File: Dockerfile (hunk 2 above)
Comment (dabeat) [reply Dockerfile]: In this case we don't need the same kind of image as in a regular service. This image is just for developing.

Are you suggesting the `RUN` commands are not necessary here?

**Pull request merged**